src/pages-and-resources/pages/PageCard.jsx

```diff
@@ -33,6 +34,34 @@ function PageCard({
     history.push(`${pagesAndResourcesPath}/${page.id}`);
   };
 
+  const settingsButton = () => {
+    if (page.hasFrontendSupport) {
+      return (
+        <IconButton
+          className="mb-0 mr-1"
+          src={Settings}
+          iconAs={Icon}
+          size="inline"
+          alt={intl.formatMessage(messages.settings)}
+          onClick={() => history.push(`${pagesAndResourcesPath}/${page.id}/settings`)}
+        />
+      );
+    }
```

Comment:
I don't think this is needed. If the page has a legacyLink, show that, if it doesn't show the settings button based on the existing logic.